provider/terraform/custom_code.rb

```diff
@@ -80,9 +99,21 @@ class Examples < Api::Object
       attr_reader :skip_test
 
       def config_documentation
+        docs_defaults = {
+          PROJECT_NAME: '<my-project-name>',
```

Comment:
I would either use flat names w/o `<` or `>` or use `{{` and `}}` for consistency with import. My preference is for flat names.

Reply:
Imo if we had an "example style guide", it would say tghat project/zone/region should always be inferred  from the provider unless we're explicitly using a second project or alternate location, in which case we'll need it defined in the config anyways.

That leaves org_id as the only important value - I think that including a well formed but obviously dummy id (all 0s, incrementing 1->9 etc) is appropriate there.

Reply:
Done.